docs/tutorial/troubleshooting/migrate_pool_by_migrating_disks.md

```diff
@@ -0,0 +1,314 @@
+# Pool Migration when underlying disks were migrated to different node
+
+## Intro
+- There can be situations where your Kubernetes cluster scales down and scales up again(To save cost). This can cause nodes to come with a different hostName and node name than the previous existing nodes, meaning the nodes that have come up are new nodes and not the same as previous nodes that earlier existed.
+- When the new nodes come up, the disks that were attached to the older nodes now get attached to the newer nodes.
+
+Consider and example where you have following nodes in a Kubernetes cluster with disks attached it:
+ 
+    Worker-1 (bd-1w1, bd-2w1 are attached to worker1) 
+  
+    Worker-2 (bd-1w2, bd-2w2 are attached to worker 1)
+  
+    Worker-3 (bd-3w3, bd-3w2 are attached to worker 1)
+
+**NOTE**: Disks attached to a node are represented by a blockdevice(bd) in OpenEBS installed cluster. A block device is of the form `bd-<some-hash>`.
+      For example, bd-1w1, bd-3w2 etc are BD resources. For illustration purpose hash is not included in BD name, e.g. `bd-1w1` represents a block device attached to worker-1.
+
+## What happens if Node Replacement Occurs?
+  If node replacement occurs in your Kubernetes cluster then cStor pool manager pods will be in pending state and the pools and volumes will go offline. Workloads using those cStor volumes will not be able to perform read and write operations on the volume.
+
+## How can this be fixed?
+  We can perform a few manual [steps](#steps-to-bring-cstor-pool-back-online) to recover from this situation. But before we do this tutorial will illustrate a Node Replacement situation. So essentially we are trying to do the following: 
+
+**__Migrate CStorPool when nodes where replaced with new nodes but same disks were reattached to the new nodes__**
+
+## Reproduce the Node Replacement situation?
+
+#### In cloud environment
+  Able to replace the nodes by deleting the node from the K8s cluster managed autoscale groups(ASG).
+
+#### On-Premise
+  Detach the disk from the node where the pool is running and attach to the different node where corresponding CSPC pool managers are not running(This might be case where resources got exhausted on the node and node was unable to perform operations).
+
```

Comment:
In this case -- if the node is not shut down -- behaviors are little different. For example, pool manager pods will still be in the running state.
Will need elaboration here.

Reply:
Updated PTAL